spec/malformed_csv.csv

```diff
@@ -0,0 +1,2 @@
+2,"23,2
+2,3,4
```

Comment:
Can you also add an example of a single really big column of floating point numbers?